python/paddle/v2/dataset/common.py

```diff
@@ -149,3 +149,47 @@ def reader():
                     yield line
 
     return reader
+
+
+def convert(output_path, eader, num_shards, name_prefix):
+    import recordio
+    import cPickle as pickle
```

Comment:
Python 2.7 and Python 3.4 include the pickle and cPickle modules already. But I didn't find official answer, there is a [answer](https://askubuntu.com/questions/755431/how-to-install-the-cpickle-module)